appservice/websocket.go

```diff
@@ -289,9 +289,11 @@ func (as *AppService) consumeWebsocket(stopFunc func(error), ws *websocket.Conn)
 			as.websocketRequestsLock.RUnlock()
 		} else {
 			as.Log.Debugfln("Received command request %s %d", msg.Command, msg.ReqID)
+			startTime := time.Now()
 			as.websocketHandlersLock.RLock()
 			handler, ok := as.websocketHandlers[msg.Command]
 			as.websocketHandlersLock.RUnlock()
+			duration := time.Now().Sub(startTime)
```

Comment:
This looks like the wrong place, handling only happens when `handler(msg.WebsocketCommand)` is called

Reply:
🤦 too early! https://github.com/mautrix/go/pull/85/commits/46e905fb56fdf7366dc4bdd7d1bffa5485940f02